src/integrations/admin/workouts-integration.php

```diff
@@ -236,17 +261,60 @@ private function get_workouts_option() {
 	 * @return string The notification to update Premium.
 	 */
 	private function get_update_premium_notice() {
-		$url = \wp_nonce_url( \self_admin_url( 'update.php?action=upgrade-plugin&plugin=wordpress-seo-premium/wp-seo-premium.php' ), 'upgrade-plugin_wordpress-seo-premium/wp-seo-premium.php' );
-
-		$notice = new Notice_Presenter(
-			\__( 'Update to the latest version of Yoast SEO Premium', 'wordpress-seo' ),
-			\sprintf(
-			/* translators: 1: Link start tag to the page to update Premium, 2: Link closing tag. */
-				__( 'It looks like you\'re running an outdated version of Yoast SEO Premium, please %1$supdate to the latest version%2$s to gain access to our updated workouts section.', 'wordpress-seo' ),
+		if ( $this->has_premium_subscription_expired() ) {
+			/* translators: %s: expands to 'Yoast SEO Premium'. */
+			$title = \sprintf( \__( 'Renew your subscription of %s', 'wordpress-seo' ), 'Yoast SEO Premium' );
+			$url   = $this->get_upsell_link();
```

Comment:
We could possibly move the `$url` outside of the if statements as it is the same for all scenarios, namely `$this->get_upsell_link()`

Reply:
Sure!